File: rust-sass-libsass-tests/ecosystem/smokes/php/smoke.php
<?php
$sass = new Sass();
$version = $sass->getLibraryVersion();
if ($version !== '3.6.6') { fwrite(STDERR, "version: $version\n"); exit(1); }
$css = $sass->compile('$color: red; .foo { color: $color; }');
if ($css !== ".foo {\n  color: red; }\n") { fwrite(STDERR, "got: [$css]\n"); exit(1); }
echo "php SMOKE-OK\n";

File: rust-sass-libsass-tests/ecosystem/smokes/nim/smoke.nim
import sass
import sass/libsass

doAssert $version() == "3.6.6", $version()
doAssert $language_version() == "3.5", $language_version()
let css = compile("$color: red; .foo { color: $color; }")
doAssert css == ".foo {\n  color: red; }\n", repr(css)
echo "nim SMOKE-OK"
